voluncheer/jobboard/tests.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+from django.test import Client
+from . import urls,views
```

Comment:
Please import using absolute path rather than relative and only put one import one each line.

```suggestion
from jobboard import urls
from jobboard import views
```

Finally, let's put django imports in a "group" and local imports in another "group" separated by a blank new line. Like so:

```suggestion
from django.test import Client
from django.test import TestCase
from django.urls import reverse

from jobboard import urls
from jobboard import views
```

This makes it easier to see what is local and what is externally provided.